tool_quality_checks/importer.py

```diff
@@ -6,18 +6,83 @@
 import logging
 from osgeo import ogr
 from os.path import basename
+from OASDGLDatachecker.tool_quality_checks.correct_import_file import (
+    correct_vector_layer,
+)
 
 logger = logging.getLogger(__name__)
 ogr.UseExceptions()
 
 
-def copy2pg_database(settings, in_filepath, layer_name, schema="public"):
+def importer(db, settings):
+    """
+        Loads your input files into the database for checks
+    """
```

Comment:
Ik weet niet waar deze vage inspringende stijl van docstrings vandaan komt?
(En: niet "loads" maar "load", als bevel)

```suggestion
    """
    Load input files into the database for checks
    """
```

(Of alles op één regel ofzo)